### ISSUE TITLE
ARM support

### ISSUE THREAD

**Issue by kostrahb** [Odania-IT/aws-kubectl#4]
Hi, would it be possible to support also ARM architecture? We would like to cost optimize and migrate to AWS graviton instances, but this image does not support arm.

**Comment by connoranderson** [comment]
Yeah this would be great. I'm also happy to contribute a PR if you can point me to how this gets deployed to dockerhub!

**Comment by kostrahb** [comment]
Meanwhile, you can use https://hub.docker.com/r/marcusveloso/aws-kubectl, which supports ARM architecture and has kubectl version v1.21.2
